hw/top_earlgrey/ip_autogen/rv_plic/doc/checklist.md

```diff
@@ -251,16 +251,16 @@ Issues        | [NO_ISSUES_PENDING][]             | Not Started |
 Review        | Reviewer(s)                       | Not Started |
 Review        | Signoff date                      | Not Started |
 
-[DESIGN_DELTAS_CAPTURED_V3]:     ../../../../doc/project_governance/checklist/README.md#design_deltas_captured_v3
-[X_PROP_ANALYSIS_COMPLETED]:     ../../../../doc/project_governance/checklist/README.md#x_prop_analysis_completed
-[FPV_ASSERTIONS_PROVEN_AT_V3]:   ../../../../doc/project_governance/checklist/README.md#fpv_assertions_proven_at_v3
-[SIM_NIGHTLY_REGRESSION_AT_V3]:  ../../../../doc/project_governance/checklist/README.md#sim_nightly_regression_at_v3
-[SIM_CODE_COVERAGE_AT_100]:      ../../../../doc/project_governance/checklist/README.md#sim_code_coverage_at_100
+[DESIGN_DELTAS_CAPTURED_V3]:     ../../../../../doc/project_governance/checklist/README.md#design_deltas_captured_v3
+[X_PROP_ANALYSIS_COMPLETED]:     ../../../../../doc/project_governance/checklist/README.md#x_prop_analysis_completed
+[FPV_ASSERTIONS_PROVEN_AT_V3]:   ../../../../../doc/project_governance/checklist/README.md#fpv_assertions_proven_at_v3
+[SIM_NIGHTLY_REGRESSION_AT_V3]:  ../../../../../doc/project_governance/checklist/README.md#sim_nightly_regression_at_v3
+[SIM_CODE_COVERAGE_AT_100]:      ../../../../../doc/project_governance/checklist/README.md#sim_code_coverage_at_100
 [SIM_FUNCTIONAL_COVERAGE_AT_100]:../../../../doc/project_governance/checklist/README.md#sim_functional_coverage_at_100
```

Comment:
```suggestion
[SIM_FUNCTIONAL_COVERAGE_AT_100]: ../../../../doc/project_governance/checklist/README.md#sim_functional_coverage_at_100
```

As above, this missing space causes the `README` -> `index` preprocessor to fail.

Reply:
done